quickannotator/api/v1/image/routes.py

```diff
@@ -50,29 +55,187 @@ def delete(self, args):
         return 204
 
 #################################################################################
-@bp.route('/<int:project_id>/search', endpoint="image_search")
-class ImageSearch(MethodView):
-    @bp.arguments(server_models.SearchImageArgsSchema, location='query')
-    @bp.response(200, server_models.ImageRespSchema(many=True))
-    def get(self, args, project_id):
-        """     returns a list of Images
-        """
-        result = db_session.query(db_models.Image).filter(db_models.Image.project_id == project_id).all()
-        return result, 200
-
+    @bp.route('/<int:project_id>/search', endpoint="image_search")
+    class ImageSearch(MethodView):
+        @bp.arguments(server_models.SearchImageArgsSchema, location='query')
+        @bp.response(200, server_models.ImageRespSchema(many=True))
+        def get(self, args, project_id):
+            """     returns a list of Images
+            """
+            result = db_session.query(db_models.Image).filter(db_models.Image.project_id == project_id).all()
+            return result, 200
+    @bp.route('/project/<int:project_id>', endpoint="images")
+    class ImageSearch(MethodView):
+        @bp.arguments(server_models.SearchImageArgsSchema, location='query')
+        @bp.response(200, server_models.ImageRespSchema(many=True))
+        def get(self, args, project_id):
+            """     returns a list of Images
+            """
+            # images = db.session.query(qadb.Image).filter(qadb.Image.project_id == project_id).all()
+            # if images is not None:
+            #     return images
+            # else:
+            #     abort(404, message="Images not found")
+            images = db_session.query(
+                *[getattr(db_models.Image, column.name) for column in db_models.Image.__table__.columns],
+                func.ST_AsGeoJSON(db_models.Image.embedding_coord).label('embedding_coord')
+            ).filter(db_models.Image.project_id == project_id).all()
+            if images is not None:
+                return images, 200
+            else:
+                abort(404, message="Image not found")
 #################################################################################
+WSI_extensions = ['svs', 'tif','dcm','vms', 'vmu', 'ndpi',
+                  'scn', 'mrxs','tiff','svslide','bif','czi']
+JSON_extensions = ['json','geojson']
+
+def import_geojson_annotation_file(image_id, annotation_class_id, gtpred, filepath):
+    '''
+    This is expected to be a geojson feature collection file, with each polygon being a feature.
+    
+    '''
+    # path = filepath.split("quickannotator/")[1]
+    with open(filepath, 'r') as file:
+        # Load the JSON data into a Python dictionary
+        data = json.load(file)["features"]
```

Comment:
probably want to use urjson or something faster? https://andrewjanowczyk.com/serialization-and-storage-of-geojson-for-in-digital-pathology/